db/migrate/20241016042459_create_users.rb

```diff
@@ -0,0 +1,18 @@
+class CreateUsers < ActiveRecord::Migration[7.2]
+  def change
+    create_table :users do |t|
+      t.string :name, null: false
+      t.string :email, null: false
+      t.string :password, null: false
+      t.string :phone, limit: 20
+      t.timestamp :date_joined, default: -> { 'CURRENT_TIMESTAMP' }
```

Comment:
what is the purpose of this attribute?

Reply:
Apu, currently we don't need this attribute as we have `created_at` attribute  the purpose of getting the joining date of a user. Removed it.